setup.cfg

```
@@ -2,7 +2,8 @@
 exclude = .eggs,ENV,build,docs/conf.py,venv
 
 [yala]
-linters=pylint,pycodestyle,isort
+radon mi args = --min C
```

```suggestion
linters=pylint,pycodestyle,isort
```

@rmotitsuki could you remove `radon` (atm we're not using `radon`) and add back the explict `linters` config? Otherwise `yala` can fallback to existing linters installed in the system.